### PR TITLE
Update cookbook_overview.rst

### PR DESCRIPTION
**Pull request by daisy20170101**
change cookbook repository LRZ gitlab

### CODE REVIEW

File: Documentation/cookbook_overview.rst [hunk 1]
```diff
@@ -1,10 +1,12 @@
 Overview
 ========
 
-This documentation is a collection of useful dynamic simulation examples to help users build models from the scratch with little efforts. Each example is demonstrated carefully with geometry building, parameter setup and result visualization. Users are suggested to repeat each example in order to get a comprehensive idea of how to set up dynamic simulation models with SeisSol. The updates of source software and news of SeisSol should be posted in SeisSol GitHub (https://github.com/SeisSol/SeisSol) or Readdocs (https://seissol.readthedocs.io/en/latest).
+This documentation is a collection of useful dynamic simulation examples to help users build models from the scratch with little efforts. Each example is demonstrated carefully with geometry building, parameter setup and result visualization. Users are suggested to repeat each example in order to get a comprehensive idea of how to set up dynamic simulation models with SeisSol. The updates of source software and news of SeisSol should be posted in SeisSol GitHub (https://github.com/SeisSol/SeisSol).
```
Comment (uphoffc) [inline Documentation/cookbook_overview.rst]: Could you (not for now but for later) put each sentence in a new line? This makes the differences between versions much easier to read. (We use the same policy also for writing papers collaboratively.)

File: Documentation/cookbook_overview.rst [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
 
 SeisSol is a part of SCEC dynamic code validation project (Harris et al. 2018) (http://scecdata.usc.edu/cvws/). Here we show several SCEC benchmarks for beginners to quickly catch up with SeisSol workflow. Each benchmark example is composed of a short problem description, a section of *geometry, initial setups (stress, nucleation, friction, etc.)*, and *simulation results*.
 
+Please note that examples used here are only for demonstration purpose. Therefore, low spatial resolution (200 m) and low intergration order (O4) is used. For detailed benchmark tests please refer to SCEC benchmark center.
```
Comment (uphoffc) [inline Documentation/cookbook_overview.rst]: intergration -> integration. I think the term "integration order" is misleading and which order is low and which is high is up to discussion ;-) I would actually delete the second sentence completely.

File: Documentation/cookbook_overview.rst [hunk 3]
```diff
@@ -49,7 +51,7 @@ Input file resources
 ~~~~~~~~~~~~~~~~~~~~
 
 The files needed to work through the examples are provided. Users may
-download freely from this repository (https://github.com/daisy20170101/SeisSol_Cookbook).
+download freely from Cookbook repository: https://gitlab.lrz.de/ru58zid/benchmarks/
```
Comment (uphoffc) [inline Documentation/cookbook_overview.rst]: The link is not available to the public.

File: Documentation/cookbook_overview.rst (hunk 2 above)
Comment (uphoffc) [reply Documentation/cookbook_overview.rst]: One could also add a link at this point: http://scecdata.usc.edu/cvws/

File: Documentation/cookbook_overview.rst (hunk 1 above)
Comment (Thomas-Ulrich) [reply Documentation/cookbook_overview.rst]: from scratch
little effort
We expect that these examples may help new users to learn how to set up dynamic simulation models with SeisSol.
I would remove 'The updates of source software and news of SeisSol should be posted in SeisSol GitHub (https://github.com/SeisSol/SeisSol).' or maybe you can explain what you mean by that

File: Documentation/cookbook_overview.rst (hunk 2 above)
Comment (Thomas-Ulrich) [reply Documentation/cookbook_overview.rst]: also 200m is not necessary low
I would rather say that the spatial resolution used may not fully resolve the dynamics

File: Documentation/cookbook_overview.rst (hunk 3 above)
Comment (Thomas-Ulrich) [reply Documentation/cookbook_overview.rst]: All files necessary to setup the cookbook examples can be downloaded at:

Comment (daisy20170101) [reply Documentation/cookbook_overview.rst]: Or I should use the old one: https://github.com/daisy20170101/SeisSol_Cookbook as the repository for all the materials?

Comment (uphoffc) [reply Documentation/cookbook_overview.rst]: No gitlab is fine, you just need to make it available to the public (Project settings -> Visibility -> Public)

File: Documentation/cookbook_overview.rst [hunk 4]
```diff
@@ -38,7 +39,7 @@ Prerequisites
 
 Before you begin any of the examples, you will need to install latest
 SeisSol from (https://github.com/SeisSol/SeisSol). The instruction can be found at https://seissol.readthedocs.io/en/latest/compilation.html. All geometry and
-tetrahedral meshing are generated using free software Gmsh (http://gmsh.info/) is needed.
+tetrahedral meshing are generated using free software Gmsh (http://gmsh.info/).
```
Comment (Thomas-Ulrich) [inline Documentation/cookbook_overview.rst]: tetrahedral meshes

**Comment by daisy20170101** [comment]
I tried, but it seems that only administrator can public a repository in gitlab? Maybe I am wrong.

I can only changed it into internal which only open to registered accounts. 


—————————————————
Dr. Duo Li
Department of Earth and Environmental Sciences
Geophysics
Ludwig-Maximilians-Universität München
Theresienstr. 41
80333 Munich 
Germany
liduoduo07@gmail.com

**Comment by uphoffc** [comment]
Yes seems so: "Public visibility has been restricted by the administrator."
In this case we have to ask LRZ or use github.

**Comment by daisy20170101** [comment]
I kept LRZ repository and added the my old github repository as a backup, until LRZ publish the main one.

**Comment by daisy20170101** [comment]
Could you verify the changes @uphoffc, if no further comments?

**Pull request merged**